<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music_player</class>
 <widget class="QMainWindow" name="music_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>503</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>music_player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="progressSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>129</width>
      <height>59</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QLabel" name="songLength">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>inf</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="value">
        <number>20</number>
       </property>
       <property name="sliderPosition">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QListView" name="songList">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="prevSong">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>88</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSong">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>88</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Next &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repeat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhello">
    <property name="title">
     <string>hello</string>
    </property>
   </widget>
   <addaction name="menuhello"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
